packages/dnsmasq.rb

```diff
@@ -0,0 +1,27 @@
+require 'package'
+
+class Dnsmasq < Package
+  description 'Lightweight, easy to configure DNS forward ,TFTP server and DHCP server'
+  homepage 'http://www.thekelleys.org.uk/dnsmasq/doc.html'
+  version '2.80'
+  source_url 'http://thekelleys.org.uk/dnsmasq/dnsmasq-2.80.tar.xz'
+  source_sha256 'cdaba2785e92665cf090646cba6f94812760b9d7d8c8d0cfb07ac819377a63bb'
+
+  depends_on 'glibc'
+  depends_on 'gmp'
+  depends_on 'libidn2'
+  depends_on 'nettle'
+  #depends_on 'make' => :build
```

Comment:
All these dependencies are included in Chromebrew core so no need to include them here.

Reply:
thx, I am new .
Next time I will get rid of these lines.